compose/animation/animation-core/build.gradle

```diff
@@ -114,12 +115,26 @@ if (AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 dependsOn(jsWasmMain)
             }
 
+            jbMain {
+                dependsOn(commonMain)
+                dependencies {
+                    implementation(libs.atomicFu)
+                }
+            }
+            jvmMain.dependsOn(jbMain)
```

Comment:
1. `jsMain.dependsOn(jbMain)` and `nativeMain.dependsOn(jbMain)` are unnecessary, as `jsNativeMain` covers them both.
2. as `jvmMain` covers Android, and we decided to not include it until we upstream, we should define only `desktopMain`

So, the result code will be:
```
desktopdMain.dependsOn(jbMain)
jsNativeMain.dependsOn(jbMain)
```

Reply:
Why not use jvmMain? It will make an easier upstreaming. You won't need to stop and think (or more likely just merge with androidMain+desktopMain).0